src/script/components/MessagesList/Message/ContentMessage/asset/TextMessageRenderer/TextMessageRenderer.tsx

```diff
@@ -121,22 +121,36 @@ export const TextMessageRenderer: FC<TextMessageRendererProps> = ({
       forwardEvent(event.nativeEvent, 'mention', mentionMsgDetails);
     }
   };
+  const extraClasses = showFullText ? 'message-quote__text--full' : '';
 
   return (
-    // We will register the click event on the paragraph element and determine the type of the element clicked.
-    // This is because the paragraph element is fed with raw HTML and we cannot register the click event on the clickabled elements directly.
-    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
-    <p
-      ref={setContainerRef}
-      key={`${editedTimestamp}:${text}`}
```

Comment:
@PatrykBuniX I removed the need for a `timestamp` here (since we want the ref to be updated when the text changes, so only a hash of the text is enough for us :) )